library/term/src/win.rs

```diff
@@ -92,24 +93,30 @@ fn bits_to_color(bits: u16) -> color::Color {
     color | (u32::from(bits) & 0x8) // copy the hi-intensity bit
 }
 
+fn get_stdout_handle() -> HANDLE {
```

Comment:
I think this function is already available from the standard library: `std::io::stdout().as_raw_handle()`